src/GitVersionCore.Tests/ConfigProviderTests.CanWriteOutEffectiveConfiguration.approved.txt

```diff
@@ -9,18 +12,21 @@ branches:
     increment: Patch
     prevent-increment-of-merged-branch-version: true
     track-merge-target: false
+    commit-message-incrementing: Enabled
```

Comment:
Do we need this to be configurable per branch?

The default should be set at the root level, you can leave it configurable per branch but it should be an override to whatever is specified in the root